src/components/UI/docs/LinkList.tsx

```diff
@@ -1,35 +1,56 @@
 import { FC } from 'react';
 import { Link, Stack, Text } from '@chakra-ui/react';
 import NextLink from 'next/link';
+import { useRouter } from 'next/router';
 
 import { NavLink } from '../../../types';
 
 interface LinksListProps {
   links: NavLink[];
 }
 
-export const LinksList: FC<LinksListProps> = ({ links }) => (
-  <Stack px={4}>
-    {links.map(({ id, to, items }) => {
-      return to ? (
-        <Stack key={id}>
-          <NextLink href={to} passHref key={id}>
-            <Link>
-              <Text textStyle='docs-nav-links' color={items ? 'primary' : 'body'}>
-                {id}
-              </Text>
-            </Link>
-          </NextLink>
-          {items && <LinksList links={items} />}
-        </Stack>
-      ) : (
-        <Stack key={id}>
-          <Text textStyle='docs-nav-links' color={items ? 'primary' : 'body'}>
-            {id}
-          </Text>
-          {items && <LinksList links={items} />}
-        </Stack>
-      );
-    })}
-  </Stack>
-);
+export const LinksList: FC<LinksListProps> = ({ links }) => {
+  const router = useRouter();
+  const { slug } = router.query;
+  return (
+    <Stack px={4}>
+      {links.map(({ id, to, items }) => {
+        const split = to?.split('/')
+        const isActive = slug && split && split[split.length - 1] === slug[slug.length - 1];
+        return to ? (
+          <Stack key={id} _hover={{ background: 'primary', color: 'bg' }} data-group>
+            <NextLink href={to} passHref key={id}>
+              <Link textDecoration='none !important'>
+                <Text
+                  textStyle='docs-nav-links'
+                  color={items || isActive ? 'primary' : 'body'}
+                  _before={{
+                    content: '"■"',
```

Comment:
you can add some vertical align to align the char with the text

```suggestion
                    content: '"■"',
                    verticalAlign: '-1.5px',
```

Reply:
@nhsz Does this need to be adjusted? I should be aligned with the baseline
<img width="775" alt="image" src="https://user-images.githubusercontent.com/54227730/205399642-22020062-f381-444a-be1a-bf25082c28cb.png">

Reply:
@nhsz Ah nevermind, I see it's sitting a little high, thanks!

Reply:
Hm.. When I checked this again locally it was actually sitting a little low. Adjusted up by 0.6px and got this: 
<img width="448" alt="image" src="https://user-images.githubusercontent.com/54227730/205401037-61dd04d1-8158-46fc-baaa-59278107126b.png">

But now I look on the netlify preview and it's shifted too high now

<img width="378" alt="image" src="https://user-images.githubusercontent.com/54227730/205402831-33fb2351-e8e5-4e30-8514-de438c7149e0.png">

Should we just leave this off? Not sure why the inconsistency

Reply:
As of latest commit (fd4aa82):

<img width="794" alt="image" src="https://user-images.githubusercontent.com/54227730/205468612-4a114908-ac2e-44f5-9495-dc3c71cbf110.png">

That should do it

Reply:
Thx!